AppCenter/AppCenter/Internals/Storage/MSAppCenterUserDefaultsPrivate.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSAppCenterUserDefaults ()
+
+/**
+ * Returns an array of keys to be migrated.
+ */
++ (NSDictionary *)keysToMigrate;
```

Comment:
I don't see implementation of this method. Is it supposed to return all already migrated keys for all services?